src/Sfx-Standalone/.build/tasks/pack.licensing.js

```diff
@@ -296,6 +297,9 @@ function generateThirdPartyNotice(deps, noticeFilePath) {
             fs.appendFileSync(noticeFd, "=========================================\r\n");
             fs.appendFileSync(noticeFd, utils.format("END OF {} NOTICES AND INFORMATION\r\n", dep.name));
         }
+    }
+    catch (e) {

```

Comment:
It would be better if write some thing about what's the current operation before log the error as all the build tasks are executed paralleled

Reply:
sure.